.storybook/main.js

```diff
@@ -67,20 +67,8 @@ if (shouldIncludePrivateStories) {
 }
 
 module.exports = {
-    core: {
-        builder: 'webpack5',
-    },
-
     stories,
     addons: [
-        {
-            name: '@storybook/addon-postcss',
```

Comment:
This plugin is not compatible with storybook 7

Reply:
hmmm, but we need it, there isn't any alternative?

Reply:
Discussed offline. It looks like we don't need the plugin anymore. Storybook is adding the postcss config by default